CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+# Contributing to ZendeskApiClient
+
+## Continuous Integration
+
+1. Create a branch, make required code changes, add unit tests and increment the [version number](https://github.com/justeat/ZendeskApiClient/blob/master/src/ZendeskApi.Build/ZendeskApi.Commons.props). We aim to follow [Semantic Versioning](https://semver.org/) guidelines within this library. 
+2. Create a pull request from `your_branch` -> `develop` branch
+   * This will trigger a github actions workflow named *ci*; this builds, runs unit tests, and packages the branch. Once approved, merge your PR into `develop` branch.
+3. A pull request from `develop` -> `master` branch will need to be made.
+   * This will trigger a github actions workflow named *ci* and will run the run-integration-tests job; tests are run on against a development Zendesk instance which belongs to JustEat Takeaway.
+   * When the integration tests have passed from your pull request and its been approved you can merge.
```

Comment:
Only JET engineers can merge into Master, might be worth outlining that here.

Reply:
done